tests/integration_tests/default_value/main.go

```diff
@@ -65,52 +68,62 @@ func main() {
 	}()
 
 	util.MustExec(sourceDB0, "create database mark;")
-	testGetZeroValue([]*sql.DB{sourceDB0, sourceDB1})
-	testGetDefaultValue([]*sql.DB{sourceDB0, sourceDB1})
+	var wg sync.WaitGroup
+	start := time.Now()
+	defer func() {
+		log.S().Infof("DefaultValue integration tests take %v", time.Since(start))
+	}()
+	wg.Add(2)
+	go testMultiDDLs([]*sql.DB{sourceDB0, sourceDB1}, &wg)
+	go testGetDefaultValue([]*sql.DB{sourceDB0, sourceDB1}, &wg)
+	wg.Wait()
 	util.MustExec(sourceDB0, "create table mark.finish_mark(a int primary key);")
 }
 
 // for every DDL, run the DDL continuously, and one goroutine for one TiDB instance to do some DML op
-func testGetDefaultValue(srcs []*sql.DB) {
-	runTime := time.Second * 60
+func testGetDefaultValue(srcs []*sql.DB, wg *sync.WaitGroup) {
+	defer wg.Done()
 	start := time.Now()
 	defer func() {
 		log.S().Infof("testGetDefaultValue take %v", time.Since(start))
 	}()
 
-	var total int
+	var wg2 sync.WaitGroup
 	for i, ddlFunc := range []func(context.Context, *sql.DB){
```

Comment:
```suggestion
	ddlFuncs := []func(context.Context, *sql.DB){
		modifyColumnDefaultValueDDL1, modifyColumnDefaultValueDDL2,
	}
	for i, ddlFunc := range ddlFuncs
```

Reply:
Seems no big difference.
I prefer keeping the previous one.